### PR TITLE
Bump ITK to 5.0.0

### PR DESCRIPTION
**Pull request by thewtex**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)

### CODE REVIEW

**Comment by thewtex** [comment]
@conda-forge-admin, please rerender

**Comment by thewtex** [comment]
@ericdill @jakirkham @scopatz any hints on how to achieve "don't build win32 or Python 2.7 and win64" with the comment syntax? Naive attempt here:

https://github.com/conda-forge/itk-feedstock/pull/8/commits/32662f2c25ca89e9da2c90a25f10cbf9a564476b

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 package:
   name: itk
-  version: "4.13.2"
+  version: "5.0.0"
 
 build:
   number: 0
-  skip: True  # [win32]
+  skip: True  # [win32 or win64 and py2k]
```
Comment (scopatz) [inline recipe/meta.yaml]: Maybe `win32 or (win64 and py==27)`

Comment (thewtex) [reply recipe/meta.yaml]: Smooth :-)

**Comment by jakirkham** [comment]
Sorry for the slow reply.

> any hints on how to achieve "don't build win32 or Python 2.7 and win64" with the comment syntax?

It's worth noting we don't build packages for Windows 32-bit any more. So skipping it explicitly is not needed (unless that is useful for you in some other context).

**Pull request merged**